pkg/api/v1alpha1/vspheredatacenterconfig_types.go

```diff
@@ -61,6 +61,20 @@ func (v *VSphereDatacenterConfig) ClearPauseAnnotation() {
 	}
 }
 
+func (v *VSphereDatacenterConfig) ConvertConfigToConfigGenerateStruct() *VSphereDatacenterConfigGenerate {
+	config := &VSphereDatacenterConfigGenerate{
```

Comment:
would this be better named `VsphereDatacenterConfigGenerated` with `Generated` instead of `Generate`? Maybe I'm misunderstanding

Reply:
Its named as `Generate` struct across all configs because the primary use of these structs was for the `generate clusterconfig` command. For consistency purposes, this struct is also named as `Generate` and not `Generated`.